one-ui/src/main/java/com/marklogic/hub/oneui/auth/MarkLogicAuthenticationManager.java

```diff
@@ -88,7 +90,7 @@ public Authentication authenticate(Authentication authentication) throws Authent
             hasManagePrivileges = true;
         }
         catch(ResourceAccessException ex) {
-            throw new BadCredentialsException("Cannot connect to MarkLogic at " + environmentInfo.mlHost + ". Are you sure MarkLogic is running?");
+            throw new BadRequestException("Cannot connect to MarkLogic at " + environmentInfo.mlHost + ". Are you sure MarkLogic is running?");
```

Comment:
I recommend tucking the original exception in here (as a 2nd argument to the constructor), just in case. It may be useful to at least see the message.